src/client/utils/dom.js

```diff
@@ -334,17 +347,12 @@ export function isDomElement (el) {
 }
 
 export function isElementInDocument (el, currentDocument) {
-    var doc        = currentDocument || document;
-    var curElement = el.parentNode;
+    var doc = currentDocument || document;
 
-    while (curElement) {
-        if (curElement === doc)
-            return true;
+    if (doc.contains)
```

Comment:
Not all of browsers support `document.contains` (for example, IE11), but as I've found most of browsers support `document.body.contains`. Maybe we can use it here?